schemas/v1alpha/bulk-metadata-response.json

```diff
@@ -5,9 +5,17 @@
   "description": "A JSON Schema to manage the format of API response objects",
   "type": "object",
   "required": [
+    "api-version",
     "apis"
   ],
   "properties": {
+    "api-version": {
```

Comment:
It's only the second duplication, so happy to leave it off for now, but did we want to make this its own object so we can just `$ref` it in these schemas?

Reply:
good idea, I'll wait till it happens for a third time and do it then